cmd/context.go

```diff
@@ -174,9 +196,22 @@ func readSecretValue() (string, error) {
 	}
 }
 
-func createContext(client api.ContextInterface, vcsType, orgName, contextName string) error {
-	err := client.CreateContext(vcsType, orgName, contextName)
-	return err
+//createContext determines if the context is being created via orgid or vcs and org name
+//and navigates to corresponding function accordingly
+func createContext(cmd *cobra.Command, client api.ContextInterface, args []string) error {
+	//skip if no orgid provided
+	if orgID != nil && strings.TrimSpace(*orgID) != "" && len(args) == 1 {
+		_, err := uuid.Parse(*orgID)
```

Comment:
what happens if the user inputs an invalid UUID?

Reply:
It'll just fall down to the return cmd.Help() command which shows the help menu. We tried to use the same logic as the comment [here](https://github.com/CircleCI-Public/circleci-cli/pull/690#discussion_r863687349) but I think it makes more sense just to print the error instead. 

Maybe something like `Org ID error: invalid uuid` or something?